webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityLink.vue

```diff
@@ -60,7 +60,7 @@
         tile>
         <img
           v-if="thumbnail"
-          :src="`${thumbnail}&size=250x150`"
```

Comment:
Why do you remove the size here ?

Reply:
Because it's set here now https://github.com/exoplatform/news/pull/675/files#diff-9823e5377655181c88db5d8bd4e7e988795970046ef7d49627a36ca782d4305cR31

Reply:
As you moved that in news, does it impacts images added in activity (not a news activity) ?

Reply:
Added images are in activity are attachments, and they are managed in ecms with thumbnail service. But this change was for news illustrations before